ckanext/iati_generator/actions/iati.py

```diff
@@ -352,3 +355,129 @@ def iati_resources_list(context, data_dict=None):
         "count": len(results),
         "results": results,
     }
+
+
+def generate_organization_xml(context, data_dict):
+    """
+    Generate IATI Organization XML for a given organization.
+
+    Parameters (data_dict keys):
+      - owner_org (str, required): Organization ID to generate XML for.
+      - namespace (str, optional): Namespace for the IATI file. Default: DEFAULT_NAMESPACE.
+
+    Behavior:
+      - Fetch all organization IATIFiles for the given owner_org+namespace.
+      - Download the CSVs to a temporary folder org-<namespace>.
+      - Run IatiOrganisationMultiCsvConverter.csv_folder_to_xml on that folder.
+      - Return the XML as a string and the local path of the generated file.
+    Returns:
+      dict: {
+        "success": <bool>,
+        "message": <str>,
+        "xml_content": <str> (if successful),
+        "xml_file_path": <str> (path to generated XML file),
```

Comment:
```suggestion
```

Just update the resource for the `FINAL_ORGANIZATION_FILE`. Do not return the XML content